ui/sqtt/ruler_graphics_item.cpp

```diff
@@ -14,12 +14,15 @@
  limitations under the License.
 */
 
-#include "ruler_graphics_item.h"
+#include "ui/sqtt/ruler_graphics_item.h"
```

Comment:
since you are moving the file anyways, we should probably remove the folder sqtt since we don't have any sqtt support in current Dive

Reply:
#682